components/brave_shields/browser/brave_shields_util.h

```diff
@@ -128,6 +128,8 @@ bool MaybeChangeReferrer(bool allow_referrers,
 
 ShieldsSettingCounts GetFPSettingCount(HostContentSettingsMap* map);
 ShieldsSettingCounts GetAdsSettingCount(HostContentSettingsMap* map);
+bool IsPatternValidForBraveContentType(ContentSettingsType content_type,
```

Comment:
I don't think this makes sense here, it's only ever used in BraveSiteSettingsHandler. Why isn't it just a method on that class?